inc/i2c_frame.h

```diff
@@ -13,6 +12,9 @@
  *    notice, this list of conditions and the following disclaimer in
  *    the documentation and/or other materials provided with the
  *    distribution.
+ * 3. Neither the name PX4 nor the names of its contributors may be
```

Comment:
This turns a 2-clause BSD into 3-clause BSD, which is however fine as its narrower.